c_emulator/riscv_sim.c

```diff
@@ -123,6 +123,10 @@ static struct option options[] = {
   {"report-arch",                 no_argument,       0, 'a'},
   {"test-signature",              required_argument, 0, 'T'},
   {"signature-granularity",       required_argument, 0, 'g'},
+  {"enable-experimental-extensions", no_argument,    0, 'X'}, // follows naming convention of LLVM
+  {"enable-Smepmp",               no_argument,       &rv_enable_Smepmp, 0 },
```

Comment:
Keep them lowercase

Reply:
With regards to -X,  we already have a -x switch.   -X should suffice for this functionality.

WIth regards to Smepmp and Zicond,  the ISA string will use these names.  I would like the publicly
exposed mnemonics to keep with the ISA string mnemonics.  I realize that this is in conflict with other
extension,  but I think that the other extension enables should line up with the ISA string.

Reply:
ISA strings are lowercase in toolchains. They are only uppercase when you use them in text as a noun (comment, diagnostic, prose, ...). You write -march=rv64gczicond, not -march=RV64GCZicond, but the error messages will talk about 'Zicond', and the specs call it that. Flags are normally lowercase, and that is consistent with -march, so keep them that way. Same goes for variables.